test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go

```diff
@@ -669,3 +733,12 @@ func countHugepagesOnNode(node *corev1.Node, sizeInMb int) (int, error) {
 	}
 	return count, nil
 }
+
+func getUpdatedNodes() []corev1.Node {
```

Comment:
what's the benefit of helper functions used once? I'd inline back this code in the calling site if we don't plan to reuse it (and sometimes even if we plan to reuse it, depending on the code flow)

Reply:
its called twice, extracted from here:
https://github.com/openshift/cluster-node-tuning-operator/blob/0686c0a24caffb32567dbb516d0b847cc613b933/test/e2e/performanceprofile/functests/2_performance_update/updating_profile.go#L76

Reply:
ok

Reply:
"[fromanirh](https://github.com/fromanirh) [8 minutes ago](https://github.com/openshift/cluster-node-tuning-operator/pull/360#discussion_r869123175)

so, are we sure this is not needed? what provides this guarantee? (applies also above)"

I've answered before. If MCP is in stated updated, the node is ready. There are similar snippets in other tests.